docs/root/operations/admin.rst

```diff
@@ -205,6 +205,12 @@ modify different aspects of the server:
   For example, get the names of all active dynamic clusters with
   ``/config_dump?resource=dynamic_active_clusters&mask=cluster.name``
 
+  .. warning
+    Passing an invalid FieldMask, such as one that indexes through a repeated field of the
+    given resource, will crash Envoy.
```

Comment:
can't we just change the implementation to echo back an error message to the admin console? This seems like annoying behavior to document.

Reply:
+1 let's please just fix the bug.

Reply:
Ack, will revert this PR and mail a proper fix.